copier/config/factory.py

```diff
@@ -1,11 +1,14 @@
 from typing import Tuple
 
+import semver
 from plumbum import local
 from plumbum.cmd import git
 
+import copier
```

Comment:
Avoid this, do a relative import instead please.

Reply:
Absolute imports are recommended in PEP8, but I guess this is to maintain consistency across the project?

Reply:
Also, I'm getting this:

```
from .. import __version__, vcs
E   ImportError: cannot import name '__version__' from partially initialized module 'copier' (most likely due to a circular import)
```

Reply:
Oh yes, of course. This is due to how that poetry plugin hacks the `__version__`.

I guess you'll be safer lazy-loading that:

```py
def copier_version() -> str:
    from .. import __version__
    return __version__
```

Reply:
Do you really want to add this function, in `tools` maybe, or do I simply add `from .. import __version__` in the `verify_minimum_version` function?